AngularNoNgAttributesSnapshotSerializer.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const jestDOMElementSerializer = require('pretty-format').plugins.DOMElement;
+
+const attributesToRemovePatterns = ['ng-reflect', '_nghost', '_ngcontent', 'ng-version'];
+
+const serialize = (node, ...rest) => {
+  const nodeCopy = node.cloneNode(true);
+  Object.values(nodeCopy.attributes)
+    .map(attribute => attribute.name)
+    .filter(attributeName =>
+      attributesToRemovePatterns
+        .some(removeAttributePattern => attributeName.startsWith(removeAttributePattern))
```

Comment:
how about extracting the filtering function out? it's the same as lines 23:25. We could also get rid of the extra `map` as well:

```js
const hasAttributesToRemove = attribute => 
  attributesToRemovePatterns.some(pattern => attribute.name.startsWith(pattern))
```

then:

```js
const serialize = (node, ...rest) => {
  // ...
    Object.values(nodeCopy.attributes)
      .filter(hasAttributesToRemove)
  // ...
}

const serializeTestFn = (val) => 
  val.attributes !== undefined && Object.values(val.attributes)
    .some(hasAttributesToRemove)

```

Reply:
Done